packages/components/psammead-caption/src/index.jsx

```diff
@@ -10,20 +12,21 @@ import {
   GEL_GROUP_4_SCREEN_WIDTH_MIN,
 } from '@bbc/gel-foundations/breakpoints';
 import {
-  GEL_LONG_PRIMER,
+  getLongPrimer,
   GEL_FF_REITH_SANS,
 } from '@bbc/gel-foundations/typography';
 
 import { C_CLOUD_DARK } from '@bbc/psammead-styles/colours';
+import { latin } from '@bbc/gel-foundations/scripts';
```

Comment:
I'm interested in figuring out if this imports `latin` even when the default isn't used. If it does always import it we should move the responsibility of passing the script to the container and make `script` a required prop.

Reply:
Yep, you are right 👍🏻